gobblin-service-loadbalancer/src/main/java/org/apache/gobblin/service/loadbalancer/GobblinServiceLoadBalancer.java

```diff
@@ -0,0 +1,57 @@
+package org.apache.gobblin.service.loadbalancer;
+
+import com.google.common.base.Optional;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.apache.gobblin.runtime.app.ApplicationException;
+import org.apache.gobblin.runtime.app.ApplicationLauncher;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class GobblinServiceLoadBalancer implements ApplicationLauncher {
+  private Server server;
```

Comment:
Try if we can merge Load Balancing into the `nginx` server container. We might need create a load balancing image on top of an nginx image.